contracts/openst-payments/PriceOracleInterface.sol

```diff
@@ -37,15 +37,6 @@ contract PriceOracleInterface {
     /*
      * Functions
      */
-    /// @dev use this function to update oracle price
-    /// @param _price price
-    /// @return expirationHeight
-    function setPrice(
```

Comment:
why we are removing function setPrice from interface?

Reply:
Interfaces serve different purposes. In this case, to the extent we think this interface is for the consumer, setting the price is not a function the consumer will need: https://github.com/OpenSTFoundation/openst-payments/pull/58#discussion_r169081791